CHANGELOG.md

```diff
@@ -95,6 +95,7 @@
 
 ## Changes since v7.6.0
 
+- [#2019](https://github.com/oauth2-proxy/oauth2-proxy/issues/2019) Refresh Token on Demand
```

Comment:
This will need to be moved to the correct location before we merge please

Reply:
Fixed